docs/resources/Application.md

```diff
@@ -23,7 +23,7 @@
 | primary_sku_id?        | snowflake                                                  | if this application is a game sold on Discord, this field will be the id of the "Game SKU" that is created, if exists      |
 | slug?                  | string                                                     | if this application is a game sold on Discord, this field will be the URL slug that links to the store page                |
 | cover_image?           | string                                                     | the application's default rich presence invite [cover image hash](#DOCS_REFERENCE/image-formatting)                        |
-| flags                  | integer                                                    | the application's public [flags](#DOCS_RESOURCES_APPLICATION/application-application-flags)                                        |
+| flags?                 | integer                                                    | the application's public [flags](#DOCS_RESOURCES_APPLICATION/application-application-flags)                                        |
 
```

Comment:
```suggestion
** \* Not included if using [Get /oauth2/applications/@me](#get-current-bot-application-information--get-oauth2applicationsme)
```

Reply:
that's already documented

![image](https://user-images.githubusercontent.com/11778454/121268016-f4de4880-c871-11eb-98ad-bad762ae0e16.png)

Reply:
Still wouldn't hurt to include that note imo

Reply:
Fair point and noticed. The goal of the extra note would be to clarify why it's optional, since almost always `flags` is returned. But that could be me :)